test/OPBuilder.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { IVerifyCallbackArgs, IVerifyCredentialResult } from '@sphereon/wellknown-dids-client';
```

Comment:
we have a couple of builder tests already in the `OP.request.spec.ts` please move these to that test suite

Reply:
fixed